roles/wazo_router_confd/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Install wazo-router-confd
+  apt:
+    name: wazo-router-confd
+    state: latest
```

Comment:
replace latest with something like {{ confd_version }}

Reply:
the idea here is not to change from what was done before the change...